node/tests/RedisClient.test.ts

```diff
@@ -32,14 +32,30 @@ const TIMEOUT = 50000;
 describe("RedisClient", () => {
     let testsFailed = 0;
     let cluster: RedisCluster;
-    let port: number;
+    let client: RedisClient;
     beforeAll(async () => {
-        cluster = await RedisCluster.createCluster(false, 1, 1);
-        port = cluster.ports()[0];
+        const args = process.argv.slice(2);
+        const standaloneEndpoints =
+            args.find((arg) => arg.startsWith("--standalone-endpoints=")) || "";
```

Comment:
Instead of having standalone endpoints and cluster endpoints, use single endpoints option and add cluster mode flag, as we do everywhere else